### ISSUE TITLE
Airport card compatibility

### ISSUE THREAD

**Issue by colin-marshall** [theracermaster/Gigabyte-GA-Z77X-DSDT-Patch#86]
Not an issue, but more of a question. I am interested in putting an airport card with the PCI-E adapter in my GA-Z77X-UD5H for native bluetooth and wifi. I was wondering what model cards and what PCI-E adapters people have used with the GA-Z77X series that they have had success with? The only requirements I have are that the wifi supports AC and it works with Sierra (and hopefully High Sierra too but that's not as crucial).

**Comment by mickelsn** [comment]
Use one of the cards from this site. Has worked great for me...

http://www.osxwifi.com/adapters

**Comment by colin-marshall** [comment]
@mickelsn thanks for the response! I'm aware of that site. Would prefer one of the ones from elsewhere that are half the price, but for sanity the extra cost is probably worth it. The cheaper ones seem to be more of a gamble.

The Fenvi FV-T919 is supposed to work great, but it's hard to find now.

**Comment by vitordino** [comment]
For wifi I use TP-Link's `TL-WDN4800` and it worked plug'n'play like a charm. but it doesn't have bluetooth ):

**Comment by colin-marshall** [comment]
@vitordino which model is your mobo and what slot do you have the card in? I currently have that card but I have some issues with it, as described here:
https://www.tonymacx86.com/threads/tp-link-wdn4800-cant-find-networks-after-a-day-or-two-of-uptime.240722/

Not sure if it's a BIOS issue, faulty WDN4800 card, wrong PCI-E slot, or something macOS related. It doesn't help that nobody else seems to have my exact problem.

**Comment by vitordino** [comment]
sorry @colin-marshall i haven't seen your comment.
have you fixed your card?

my model is GA-Z77X-D3H, i have an [repo](https://github.com/vitordino/MacOS-GA-Z77X-D3H) that i keep my `EFI` folder

**Comment by colin-marshall** [comment]
@vitordino no problem. I ditched the WDN4800 and bought a Broadcom card off of eBay with a PCI-E adapter. Bluetooth worked right away but WiFi would disable seconds after turning it on. It ended up being not a genuine Apple card (even thought it was stated to be one) but the seller was cool and accepted a return.

I went back to the WDN4800 for a short period of time before I read up on how to re-route the cat5e in my home so that I could use ethernet on my Hackintosh and really wish I had just done that in the first place.
